### DISCUSSION TITLE
How to update update private dependency?

### DISCUSSION THREAD

**Discussion by Fatin1234567** [renovatebot/renovate#19150] [Request Help] [answered]
As described at [https://docs.renovatebot.com/getting-started/private-packages/#nuget](url) , I added these to config.js (renovate-runner)

```
module.exports = {
  labels: ["dependencies"],
  hostRules: [
    {
      hostType: 'nuget',
      matchHost: 'https://<>/api/v4/groups/17/-/packages/nuget/',
      username: 'none',
      password: '<>',
    }

  ],

   repositories: ['fatin.kazi/testrenovate1']


};

```

![image](https://user-images.githubusercontent.com/78020343/204770186-ff0b14c8-699a-4fbf-9b7c-1c33719b8d46.png)

Im getting a 404, guessing its trying to use default 

I get this under the issue
⚠ Dependency Lookup Warnings ⚠
Renovate failed to look up the following dependencies: rdztest, , A70.

Can you help me understand what am I missing?

**Comment by rarkins** [comment]
You haven't included enough info about what you're trying to do nor what you are seeing going wrong, but I think your `matchHost` is far too long. Set it to the minimum URL which covers all the package downloads, such as `matchHost: "https://<>/api/v4/groups/17/-/packages/nuget/",`

**Reply by Fatin1234567** [reply]
@rarkins Enough info now?

**Comment by rarkins** [comment, marked as answer]
It looks like you need to configure `registryUrls` to tell Renovate which registry to look up with.

**Reply by Fatin1234567** [reply]
Cool, so if you have multiple package source then you just add to registryUrls and hostRules?

**Reply by rarkins** [reply]
Step 1 is telling renovate where to look for packages (registryUrls)

Step 2 is how to authenticate (hostRules)
